README.md

```diff
@@ -1 +1,178 @@
 # ng-bem-components
+
+An [AngularJS](angularjs.org) `angular-bem` module for using components based on [`i-bem.js`](https://bem.info/libs/bem-core/v2.7.0/desktop/i-bem/) framework.
+See [bem-core](https://github.com/bem/bem-core) and [bem-components](https://github.com/bem/bem-components) for further details.
+
+Module will allow you to render `BEMJSON`, help you to work with BEM mods and event, and provide some integrations of `i-bem` blocks with Angular directives.
+
+## Install
+
+```sh
+bower i ng-bem-components --save
+```
+
+`angular-bem` depends on `i-bem__dom`, `jquery`, `BEMHTML` and `BEMTREE` (optional) from `ymodules` modular system.
+Include `angular-bem.js` in your js bundle or html file in any place where `ymodules` already defined.
+
+## Bootstrap Angular app
+
+You will not able to use `ngApp` directive since dependencies provided asynchronously.
+Require `angular-bem` module from `ymodules` to [bootstrap](https://docs.angularjs.org/api/ng/function/angular.bootstrap) your angular app, instead.
+
+```js
+angular.module('my-app', ['angular-bem', ...]);
+
+modules.require('angular-bem', function(){
+    angular.bootstrap(document, ['my-app']);
+});
+```
+
+## Services
+
+Module provides `i-bem__dom`, `BEMHTML` and `BEMTREE` from `ymodules` as angular injectable services `bemdom`, `bemhtml` and `bemtree` respectively.
+`bemtree` service will be `undefined` (but still exists) if `BEMTREE` is not defined in ymodules.
+
+### ngbem
+
+`ngbem` service for processing `BEMJSON` with `BEMHTML` and `BEMTREE` templates.
+
+#### Dependencies
+
+`bemhtml` <br>
+`bemtree` <br>
+`$q` <br>
+`$log`
+
+#### Usage
+
+`ngbem(bemjson, useBemtree);`
+
+##### Arguments
+
+Param      | Type      | Details
+-----------|-----------|--------
+bemjson    | `Object`  | Object in `BEMJSON` format to process.
+useBemtree | `Boolean` | If set to `true` process `BEMJSON` with `BEMTREE` first of all.
+
+##### Returns
+
+`Promise` - Promise that will be resolved with HTML string processes from `bemjson`.
+
+## Directives
+
+### ngBem
+
+Directive renders provided `BEMJSON` using `BEMHTML` and optionally `BEMTREE` templates.
+`BEMJSON` has access to scope since is treated as [Angular Expression](https://docs.angularjs.org/guide/expression).
+Rendered html initialized with [`bemdom`'s `init()`](https://bem.info/libs/bem-core/v2.7.0/desktop/i-bem/jsdoc/#jsdoc-init-1) method and compiled with Angular [$compile](https://docs.angularjs.org/api/ng/service/$compile) service.
```

Comment:
just to avoid extra redirect (as soon as we already know it's English documentation)